optimum/exporters/onnx/config.py

```diff
@@ -85,7 +89,7 @@ def inputs(self) -> Mapping[str, Mapping[int, str]]:
 
         return common_inputs
 
-    def create_dummy_input_generator_classes(self):
+    def _create_dummy_input_generator_classes(self, **kwargs) -> List["DummyInputGenerator"]:
```

Comment:
None of the dummy input generators need `**kwargs` in this case?

Reply:
They do. Should be fixed in https://github.com/huggingface/optimum/pull/522/commits/4ec889317b1bb3528191632545c997b0bc0e8412 , can you check again if it looks good?